src/settings-ui/Settings.UI/Views/GeneralPage.xaml

```diff
@@ -172,12 +172,11 @@
 
 
 
-
-
                 <controls:SettingsGroup x:Uid="Admin_Mode">
                     <controls:SettingExpander IsExpanded="True">
                         <controls:SettingExpander.Header>
                             <controls:Setting x:Uid="Admin_Mode" Icon="&#xE7EF;" Description="{Binding Mode=OneWay, Path=RunningAsText}" Style="{StaticResource ExpanderHeaderSettingStyle}">
+                                <HyperlinkButton NavigateUri="https://aka.ms/powertoysDetectedElevatedHelp" x:Uid="GeneralPage_ToggleSwitch_AlwaysRunElevated_Link"/>
```

Comment:
![image](https://user-images.githubusercontent.com/9866362/183862859-78d5aafc-fe72-4b1c-ac1c-48eec914dc9f.png)

Where did this go? You have a Description set here already. The HyperLink needs to be in the Description.

Reply:
But what happens then with the {Binding Mode=OneWay, Path=RunningAsText} 🤔?

Reply:
Yeah, this is the one I actually need the feedback on. My idea is visible in my issue.

Reply:
@Jay-o-Way , Reviewing this and the issue. The idea is to put the link close to the "Running as user" / "Running as admin", right? I agree that can make sense, since Administrator mode is just spelled out twice already.
What's blocking here?

Reply:
The strings don't show as I want and the link doesn't seem to appear at all.

Reply:
Hi, but the idea is to move "running as user" to the top and have the hyperlink as description, right?

Reply:
Something like this?
![image](https://user-images.githubusercontent.com/26118718/185442950-2dabe051-e1fc-405d-a78f-9af51f08276a.png)

Reply:
What needs to be done is to get this is putting the HyperLink button inside a `<controls:Setting.Description>`, set the Header instead of the description to "RunningAsText" and change the Id of the Setting so that it doesn't get the Header from Resources.

![image](https://user-images.githubusercontent.com/26118718/185443555-1971b279-035d-4115-8b85-184615fb3edf.png)

Hope this helps.

Reply:
@Jay-o-Way , hope this helps

Reply:
Yes! You got it! ❤